From d66fbb9d12ae2e5dbaf13d23b9abb4b6bf187d7f Mon Sep 17 00:00:00 2001
From: Pavel Grunt <pgrunt@redhat.com>
Date: Thu, 4 May 2017 11:52:25 +0200
Subject: [PATCH 31/55] vdagentd: Allow to compile without session info

Add a dummy implementation of session_info_session_is_locked

configure supports --with-session-info=none, however the build fails:
 src_spice_vdagentd-vdagentd.o: In function `do_client_file_xfer':
 src/vdagentd/vdagentd.c:336: undefined reference to
                                 `session_info_session_is_locked'
Acked-by: Victor Toso <victortoso@redhat.com>
---
 src/vdagentd/dummy-session-info.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/vdagentd/dummy-session-info.c b/src/vdagentd/dummy-session-info.c
index 8ffe5e4..d8f1833 100644
--- a/src/vdagentd/dummy-session-info.c
+++ b/src/vdagentd/dummy-session-info.c
@@ -54,3 +54,8 @@ gboolean session_info_is_user(struct session_info *si)
 {
     return TRUE;
 }
+
+gboolean session_info_session_is_locked(G_GNUC_UNUSED struct session_info *si)
+{
+    return FALSE;
+}
-- 
2.16.1

